wolfssl/internal.h

```diff
@@ -2944,11 +2964,18 @@ int TLSX_EncryptThenMac_Respond(WOLFSSL* ssl);
 /* Cookie extension information - cookie data. */
 typedef struct Cookie {
     word16 len;
-    byte   data;
+    /* Ignore "nonstandard extension used : zero-sized array in struct/union"
+     * MSVC warning */
+    #ifdef _MSC_VER
+    #pragma warning(disable: 4200)
+    #endif
+    byte   data[];
```

Comment:
@douzzer , don't we have a better way to do this?

Reply:
@dgarske, @julek-wolfssl already merged in this suppression later in internal.h, in b2a80cbe65.  At a minimum, he should move that later one higher in internal.h so it's done only once.

In wolfSentry I avoid suppressions with this in `wolfsentry_settings.h`:
```
#if defined(__STRICT_ANSI__)
#define WOLFSENTRY_FLEXIBLE_ARRAY_SIZE 1
#elif defined(__GNUC__) && !defined(__clang__)
#define WOLFSENTRY_FLEXIBLE_ARRAY_SIZE
#else
#define WOLFSENTRY_FLEXIBLE_ARRAY_SIZE 0
#endif
```
but the lowest impact way forward is to keep the MSVC suppression and just move it up.

Reply:
Using the wolfSentry method would use one additional byte wouldn't it? Its not much but it can start to add up.